src/parser/test/ParserTest.cpp

```diff
@@ -715,6 +715,133 @@ TEST(Parser, DeleteEdge) {
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "$var = GO FROM 1 over e1; "
+                            "FETCH PROP ON person $var.id YIELD person.name, person.age";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON player 1,2,3 "
+                            "YIELD DISTINCT person.name, person.age";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON person uuid(\"dutor\")";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON person uuid(\"dutor\"), uuid(\"darion\")";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON person uuid(\"dutor\") "
+                            "YIELD person.name, person.age";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON person uuid(\"dutor\"), uuid(\"darion\") "
+                            "YIELD person.name, person.age";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+}
+
+TEST(Parser, FetchEdge) {
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON transfer 12345 -> -54321";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "FETCH PROP ON transfer 12345 -> -54321 "
+                            "YIELD transfer.time";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 12345 OVER transfer "
+                            "YIELD transfer.time";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "GO FROM 12345 OVER transfer "
+                            "YIELD transfer._src AS s, serve._dst AS d | "
+                            "FETCH PROP ON transfer $-.s -> $-.d YIELD transfer.amount";
+        auto result = parser.parse(query);
+        ASSERT_TRUE(result.ok()) << result.status();
+    }
+    {
+        GQLParser parser;
+        std::string query = "$var = GO FROM 12345 OVER transfer "
+                            "YIELD transfer._src AS s, edu._dst AS d; "
+                            "FETCH PROP ON serve $var.s -> $var.d YIELD transfer.amount";
```

Comment:
Same problem as mentioned above.

Reply:
This case is refer to [fetch-syntax](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/statement-syntax/data-manipulation-statements/fetch-syntax.md).  But no mistakes were found.

Reply:
I mean you are about to fetch prop on serve while yield from transfer.